core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -82,10 +82,11 @@ class ExecutorStageSummary private[spark](
     val shuffleWriteRecords : Long,
     val memoryBytesSpilled : Long,
     val diskBytesSpilled : Long,
-    val isBlacklistedForStage: Boolean,
+    val isBlacklistedForStage: Boolean, // deprecated
```

Comment:
the problem with that is that this is the rest api and we can't really deprecate it without having some new api, which would like be a v2 instead of v1 or a different endpoint. Since we don't have that I don't think it makes sense to deprecate the entire structure.  I think this is the first time this has happened as well so I'm open to ideas on how we want to handle.

Reply:
Even if we can't remove it, we can at least mark it as deprecated so that consumers are aware that they're encouraged to migrate to the new field, right? Is your concern that a deprecation without plans for removal is pointless?

Reply:
deprecation without an alternative is pointless. So I think we would need a plan on what to replace it with. @Ngone51  any thoughts on the rest api deprecation?

Reply:
Hm. Maybe we have some misunderstanding. I don't mean marking `ExecutorStageSummary` as deprecated, I mean marking `isBlacklistedForStage` as deprecated. The alternative was added as `isExcludedForStage`

Reply:
oh I see, yeah I think we can do that